src/js/components/Spinner/Spinner.js

```diff
@@ -4,15 +4,19 @@ import React, {
   useContext,
   useEffect,
 } from 'react';
-import { ThemeContext } from 'styled-components';
+import styled, { ThemeContext } from 'styled-components';
 
 import { AnnounceContext } from '../../contexts/AnnounceContext';
 import { Box } from '../Box';
 import { defaultProps } from '../../default-props';
 import { SpinnerPropTypes } from './propTypes';
 
+const SpinnerBox = styled(Box)`
+  min-width: ${(props) => props.width};
+`;
+
 const BasicSpinner = ({ ref, size, ...rest }) => (
-  <Box height={size} width={size} ref={ref} {...rest} />
+  <SpinnerBox height={size} width={size} ref={ref} {...rest} />
```

Comment:
Any reason we didn't just use `<Box flex={false} />`?

Reply:
The main reason is that I am not fully familiar with API yet. Changing `flex-shrink` to `0` via `flex={false}` will work as well and without additional styled component 👍

Reply:
also didnt think about `flex={false}` Thanks Eric